tests/unit/DateObject.ts

```diff
@@ -51,169 +49,171 @@ registerSuite({
 			object = new DateObject(date);
 		},
 
-		'year': function () {
-			assert.strictEqual(object.year, date.getFullYear());
-			date.setFullYear(object.year = 1);
-			assert.strictEqual(object.year, date.getFullYear());
-
-			object = new DateObject({ year: 2005, month: 12, dayOfMonth: 27 });
-			object.year += 1;
-			assert.strictEqual(+object, +new Date(2006, 11, 27));
-
-			object = new DateObject({ year: 2005, month: 12, dayOfMonth: 27 });
-			object.year -= 1;
-			assert.strictEqual(+object, +new Date(2004, 11, 27));
-
-			object = new DateObject({ year: 2000, month: 2, dayOfMonth: 29 });
-			object.year += 1;
-			assert.strictEqual(+object, +new Date(2001, 1, 28));
-
-			object = new DateObject({ year: 2000, month: 2, dayOfMonth: 29 });
-			object.year += 5;
-			assert.strictEqual(+object, +new Date(2005, 1, 28));
-
-			object = new DateObject({ year: 1900, month: 12, dayOfMonth: 31 });
-			object.year += 30;
-			assert.strictEqual(+object, +new Date(1930, 11, 31));
-
-			object = new DateObject({ year: 1995, month: 12, dayOfMonth: 31 });
-			object.year += 35;
-			assert.strictEqual(+object, +new Date(2030, 11, 31));
-		},
-
-		'month': function () {
-			assert.strictEqual(object.month, date.getMonth() + 1);
-			date.setMonth((object.month = 1) - 1);
-			assert.strictEqual(object.month, date.getMonth() + 1);
+		tests: {
```

Comment:
what's the purpose of the `test` wrapping object?

Reply:
I've found out, that if there are lifecycle functions such as `before` then all tests need to be wrapped in a `tests` object.